msal/application.py

```diff
@@ -852,8 +852,7 @@ def acquire_token_interactive(self,
             port=None,
             prompt=None,
             domain_hint=None,  # type: Optional[str]
-            claims_challenge=None,
-            **kwargs):
+            claims_challenge=None):
```

Comment:
Minor thing: In case you don't already know, this can be written in the following way, so that when we eventually add that `**kwargs` (or whatever new parameters), it will generate minimal red-and-green lines in the new PR.

```suggestion
            claims_challenge=None,
            ):
```